addons/website/data/website_data.xml

```diff
@@ -44,87 +44,73 @@
                 <section class="s_text_block pt40 pb40 o_colored_level " data-snippet="s_text_block">
                     <div class="container s_allow_columns">
                         <div class="row">
-                            <div class="col-lg-8 mt-4 mt-lg-0">
-                                <p>
+                            <div class="col-lg-7 mt-4 mt-lg-0">
+                                <p class="lead">
                                     Contact us about anything related to our company or services.<br/>
                                     We'll do our best to get back to you as soon as possible.
                                 </p>
                                 <section class="s_website_form" data-vcss="001" data-snippet="s_website_form">
                                     <div class="container">
                                         <form id="contactus_form" action="/website/form/" method="post" enctype="multipart/form-data" class="o_mark_required" data-mark="*" data-model_name="mail.mail" data-success-mode="redirect" data-success-page="/contactus-thank-you" data-pre-fill="true">
                                             <div class="s_website_form_rows row s_col_no_bgcolor">
-                                                <div class="mb-0 py-2 col-12 s_website_form_field s_website_form_custom s_website_form_required" data-type="char" data-name="Field">
-                                                    <div class="row s_col_no_resize s_col_no_bgcolor">
-                                                        <label class="col-form-label col-sm-auto s_website_form_label" style="width: 200px" for="contact1">
-                                                            <span class="s_website_form_label_content">Name</span>
-                                                            <span class="s_website_form_mark"> *</span>
-                                                        </label>
-                                                        <div class="col-sm">
-                                                            <input id="contact1" type="text" class="form-control s_website_form_input" name="name" required="" data-fill-with="name"/>
-                                                        </div>
+                                                <div class="mb-0 py-2 col-6 s_website_form_field s_website_form_custom s_website_form_required" data-type="char" data-name="Field">
```

Comment:
```suggestion
                                                <div class="mb-0 py-2 col-lg-6 s_website_form_field s_website_form_custom s_website_form_required" data-type="char" data-name="Field">
```
Content gets truncated almost all the times on mobile.
![image](https://github.com/user-attachments/assets/79865c09-baff-4d2f-86b5-f15ff22b86ad)

Reply:
Fixed 👍